Polygon_mesh_processing/doc/Polygon_mesh_processing/NamedParameters.txt

```diff
@@ -152,6 +152,30 @@ the property map containing information about edges of the input polygon mesh be
 a default property map where no edge is constrained is provided.
 \cgalNPEnd
 
+\cgalNPBegin{ maximum_number_of_patches } \anchor PMP_maximum_number_of_patches
+contains the surface patch beeing treated +1.\n
+\b Type : `std::size_t`* \n
```

Comment:
I probably don't understand something because it makes no sense to me. Can't this _shared_ variable be passed by reference internally so that internal calls update it. If the user need to get the last value then a return type should be used.